src/sdltiles.cpp

```diff
@@ -1381,6 +843,161 @@ bool Font::draw_window( const catacurses::window &w, const point &offset )
     return update;
 }
 
+bool draw_window( Font_Ptr &font, const catacurses::window &w )
```

Comment:
```suggestion
static bool draw_window( Font_Ptr &font, const catacurses::window &w )
```

Reply:
so that was it
I wonder why I was able to build both debug and release on clang, but the ci gcc failed ...